### ISSUE TITLE
Skin bug

### ISSUE THREAD

**Issue by RyuruKakitashi** [fluddokt/opsu#193]
I downloaded the "- Sagiri edition-" skin and founs out the settings was gone when i equiped the skin help

**Comment by fluddokt** [comment]
Yeah that skin is not currently supported, you are going to need to delete that skin or edit opsu.cfg and change "Skin" to something else or delete opsu.cfg
